documentation/wiki/Providing-Binary-Logs.md

```diff
@@ -6,9 +6,23 @@ However, you should be aware what type of information is captured in the binary
 
 ⚠ NOTE: some build environments make secrets available using environment variables. Before sharing a binary log, make sure it does not expose API tokens or other important secrets.
 
-You can create a binary log by passing the `-bl` parameter to MSBuild. You can explore the contents of the generated .binlog file using [MSBuild Structured Log Viewer](http://msbuildlog.com/) or in your browser using [Live Structured Log Viewer](https://live.msbuildlog.com). Note: We don't capture any data from binary logs viewed on your browser.
+You can create a binary log by passing the `-bl` parameter to MSBuild (`MSBuild.exe` or `dotnet build`). You can explore the contents of the generated .binlog file using [MSBuild Structured Log Viewer](http://msbuildlog.com/) or in your browser using [Live Structured Log Viewer](https://live.msbuildlog.com). Note: We don't capture any data from binary logs viewed on your browser.
 
 [More details about binary logs](Binary-Log.md)
 
 ## Capturing Binary Logs Through Visual Studio
+### (Preferred way) Capturing logs for all MSBuild invocations
+Via setting `MSBUILDDEBUGENGINE` environment variable to `'1'`:
+```
+> SET MSBUILDDEBUGENGINE=1
+> devenv.exe MySolution.sln
+```
+
+MSBuild binary logs are then captured to `MSBuild_Logs` subfolder of a current folder.
```

Comment:
I do not find this reliable and generally prefer explicitly specifying `MSBUILDDEBUGPATH`.

Reply:
(If MSBUILDDEBUGPATH is unset, it tries the current folder then temp...it's more confusing.)